src/main/java/frc/team4909/robot/commands/Linefollow.java

```diff
@@ -1,62 +1,48 @@
-package frc.team4909.robot.commands;
+package frc.team4909.robot.subsystems.drivetrain;
 
 import edu.wpi.first.wpilibj.DigitalInput;
 import edu.wpi.first.wpilibj.command.Command;
-import edu.wpi.first.wpilibj.command.InstantCommand;
 import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
 import frc.team4909.robot.Robot;
-import frc.team4909.robot.subsystems.drivetrain.DriveTrainSubsystem;
+import frc.team4909.robot.RobotConstants;
 
 public class Linefollow extends Command {
 
-    public static DigitalInput LeftSensor, MiddleLeftSensor, MiddleRightSensor, RightSensor;
-
-    private double velocity = 0.5;
+    public static DigitalInput leftSensor, middleLeftSensor, middleRightSensor, rightSensor;
 
     public Linefollow() {
-        requires(Robot.drivetrainsub);
+        requires(Robot.drivetrainSubsystem);
     }
 
     protected void initialize() {
-        LeftSensor = new DigitalInput(0);
-        MiddleLeftSensor = new DigitalInput(1);
-        MiddleRightSensor = new DigitalInput(2);
-        RightSensor = new DigitalInput(3);
+        leftSensor = new DigitalInput(0);
+        middleLeftSensor = new DigitalInput(1);
+        middleRightSensor = new DigitalInput(2);
+        rightSensor = new DigitalInput(3);
     }
 
     protected void execute() {
-        SmartDashboard.putBoolean("LEFT line following", !LeftSensor.get());
-        SmartDashboard.putBoolean("RIGHT line following", !RightSensor.get());
-
-        boolean left = MiddleLeftSensor.get();
-        boolean right = MiddleRightSensor.get();
+        SmartDashboard.putBoolean("LEFT line following", !leftSensor.get());
+        SmartDashboard.putBoolean("RIGHT line following", !rightSensor.get());
 
-        // Should work no matter which side of the robot the cargo ship is on
+        boolean left = middleLeftSensor.get(); // true = sensor is OFF line; false = sensor is ON line
+        boolean right = middleRightSensor.get(); // true = sensor is OFF line; false = sensor is ON line
```

Comment:
Good comments here describing purpose. Suggest, however, also renaming these variables to `leftOffLine` and `rightOffLine` so that it's clear, when reading the code that uses them, what their meaning is.